action.yml

```
@@ -36,6 +36,12 @@ inputs:
     description: >
       Username for docker registry detected from the repository.
     default: _json_key
+  auth-only:
+    required: true
+    description: >
+      This flag can be used in case only authentication against docker 
+      repo is needed
+    default: false
```

As this is yaml I think you should put quotes
```suggestion
    default: 'false'
```